sprite.h

```diff
@@ -45,6 +46,33 @@ class Sprite : public Identifiable<Empty> {
 	size_t m_trail_start_index;
 };
 
+class SpriteWiggler {
+public:
+	SpriteWiggler();
+	void randomize_tendency();
+	void wiggle(Context &ctx, const Point &home, Point &relpos, const double &magnitude);
+
+private:
+	template<typename T>							// first value gets multiplied or divided by 1 + random(second value)
+	using pair_randomness = std::pair<T, double>;	// probably needs a better name :)
```

Comment:
You might consider getting rid of this template alias entirely and just writing out `std::pair<double, size_t>` or whatever. It makes things a little simpler and you don't have to think of a better name :)